web/blog/from-slack-to-discord/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: From Slack to Discord
+description: Moving our community chat from Slack to Discord
+authors: mavam
+image: slack-to-discord.light.png
```

Comment:
```suggestion
image: /blog/from-slack-to-discord/slack-to-discord.light.png
```

@ditya I also tried this to get the image render in the main page carousel. Not sure if it makes more sense to just remove this metadata item, or whether there was a quick fix to make it work.

Reply:
Pushed the quick fix.

Reply:
That works, but is the idea now that we pollute `/img` for blog post images?

Reply:
I say for now, yes. An overhaul of the docs site is planned, and I say we don't spend more time on patching it before then.

Reply:
Did I miss something? What's the planned overhaul for the docs site?

Reply:
Taking this discussion offline.